mappings/net/minecraft/world/biome/BuiltinBiomes.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_5504 net/minecraft/world/biome/BuiltinBiomes
 	METHOD method_40363 bootstrap (Lnet/minecraft/class_7891;)V
 		ARG 0 biomeRegisterable
-	METHOD method_49391 (Lnet/minecraft/class_7891;)V
+	METHOD method_49391 bootstrapOneTwenty (Lnet/minecraft/class_7891;)V
 		ARG 0 biomeRegistry
```

Comment:
Are the parameter names in these two supposed to be different? (unrelated to the actual change)

Reply:
Good catch, fixed